### DISCUSSION TITLE
Questions about Outgoing/incomg calls and Currently Playing Info

### DISCUSSION THREAD

**Discussion by uzair-io** [Arkq/bluez-alsa#655] [Q&A]
Hi, I've install bluez-also on my system and it works flawlessly, however I wanted to know if there is any way to get information like if theres an incoming call or currently playing media information, or is this beyond the scpe of bluex-alsa. If it is, I would like suggestions as to how I can access any API/How I would go about using other software to get this.

**Comment by arkq** [comment]
>  get information like if there is an incoming call

Please read this wiki page (maybe it will help): https://github.com/arkq/bluez-alsa/wiki/Using-BlueALSA-with-HFP-and-HSP-Devices

> currently playing media information

Try this one (I have not tried it by myself, though): https://wiki.archlinux.org/title/MPRIS#Bluetooth

**Reply by uzair-io** [reply]
Thanks for your response. I discovered my Bluetooth device has a AVRCP interface through which I can play/pause media. I tested it using D-spy, which worked well enough until I realised Android doesn't have the best AVRCP implementation, though it should work for my needs. In terms of calls I still need to try the RFCOMM interface.
